### PR TITLE
React.PropTypes is deprecated from React 15.5

### PR DESCRIPTION
**Pull request by yosmoc**
Migrated by react-codemod + fixed trivial things by hand.

### Before

```
...
/Users/yoshi/src/github.com/yosmoc/Boostnote/browser/main/TopBar/index.js
    1:1   warning  React.PropTypes is deprecated since React 15.5.0, use the npm module prop-types instead  react/no-deprecated
    5:8   warning  'NewNoteModal' is defined but never used                                                 no-unused-vars
   10:9   warning  'dialog' is assigned a value but never used                                              no-unused-vars
   12:7   warning  'OSX' is assigned a value but never used                                                 no-unused-vars
  124:11  warning  'config' is never reassigned. Use 'const' instead                                        prefer-const
  124:19  warning  'style' is never reassigned. Use 'const' instead                                         prefer-const
  124:26  warning  'data' is assigned a value but never used                                                no-unused-vars
  124:26  warning  'data' is never reassigned. Use 'const' instead                                          prefer-const
  124:32  warning  'location' is never reassigned. Use 'const' instead                                      prefer-const
...
✖ 568 problems (0 errors, 568 warnings)
```

### After

```
...
/Users/yoshi/src/github.com/yosmoc/Boostnote/browser/main/TopBar/index.js
    6:8   warning  'NewNoteModal' is defined but never used             no-unused-vars
   11:9   warning  'dialog' is assigned a value but never used          no-unused-vars
   13:7   warning  'OSX' is assigned a value but never used             no-unused-vars
  125:11  warning  'config' is never reassigned. Use 'const' instead    prefer-const
  125:19  warning  'style' is never reassigned. Use 'const' instead     prefer-const
  125:26  warning  'data' is assigned a value but never used            no-unused-vars
  125:26  warning  'data' is never reassigned. Use 'const' instead      prefer-const
  125:32  warning  'location' is never reassigned. Use 'const' instead  prefer-const
...
✖ 527 problems (0 errors, 527 warnings)
```

### CODE REVIEW

**Comment by kazup01** [comment]
Hi @yosmoc , thanks for your support every time.
Sorry, there is a conflict. Could you fix it?

**Comment by yosmoc** [comment]
@kazup01 Resolved conflict!

**Comment by kazup01** [comment]
Thanks!

**Comment by yosmoc** [comment]
Sorry, I found my mistake. I pushed again now.

File: browser/components/RealtimeNotification.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
-import React, { PropTypes } from 'react'
+import PropTypes from 'prop-types'
```
Comment (kohei-takata) [inline browser/components/RealtimeNotification.js]: @yosmoc 
Please resolve conflict.
This line is not needed, I think.

Comment (yosmoc) [reply browser/components/RealtimeNotification.js]: fixed!

**Pull request merged**